contributors.md

```diff
@@ -2,90 +2,107 @@
-Example: <https://github.com/r-lib/gh/releases/tag/v1.4.0>.
-Pre-releases (GitHub) and upcoming releases (GitLab) are ignored to ensure each release has the full endorsement of its maintainer.
-:::
-
-::: {.callout-note collapse="true"}
-## Packages in a GitHub repo subdirectory
-In rare cases, the package may be in a subdirectory of a GitHub repo.
-In these cases, your text file may instead contain a JSON list with fields `package`, `url`, `subdir`, and `branch`, and the `branch` field must be `"*release"`.
-[Example](https://github.com/r-multiverse/contributions/blob/main/packages/arrow):
-
-```json
-{
-  "package": "arrow",
-  "url": "https://github.com/apache/arrow",
-  "subdir": "r",
-  "branch": "*release"
-}
-```
-:::
+### Package listings
 
-## How to create a badge
+A package listing is a text file in <https://github.com/r-multiverse/contributions/tree/main/packages>
+which points to the home of the package on GitHub/GitLab.
+The file name of each listing is the name of the package.
+In most cases, the file itself contains the package URL on GitHub or GitLab ([example](https://github.com/r-multiverse/contributions/blob/main/packages/polars)).
+But if the package is a subdirectory of the GitHub/GitLab repository,
+you may instead write custom JSON list with fields `package`, `url`, `subdir`, and `branch: "*release"` ([example](https://github.com/r-multiverse/contributions/blob/main/packages/arrow)).
 
-[<img src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2Fmirai&query=%24.Version&label=r-multiverse" alt="R-multiverse status" />](https://community.r-multiverse.org/mirai)
+### Pull request review
 
-To add a dynamic 'R-multiverse' badge for package readme files, like the one above, copy the following markdown snippet, replacing 'pkgNAME' with the actual package name in both places it appears:
+An automated process periodically scans and reviews each new pull request to <https://github.com/r-multiverse/contributions>.
+Depending on the automated checks, the bot either merges the pull request
+or flags it for manual review by a [moderator](governance.md#moderator).
+Some moderators only review packages specific to a subject matter area,
+so please consider mentioning a [topic](https://r-multiverse.org/topics/index.html)
+in the title of the pull request.
 
-```md
-[![R-multiverse status](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fcommunity.r-multiverse.org%2Fapi%2Fpackages%2FpkgNAME&query=%24.Version&label=r-multiverse)](https://community.r-multiverse.org/pkgNAME)
-```
+## Community
+
+After registration, the latest release of the package will
+automatically appear in the [Community](community.md) repository.
+[GitHub Actions](https://github.com/features/actions) workflows periodically
```

Comment:
In this paragraph it seems the line breaks are not semantic. (I'm still working on https://github.com/maelle/aeolus, in a few weeks it should be usable)

Reply:
I'll keep this in mind for future PRs.